modules/stochastic_tools/test/tests/samplers/morris/gold/morris5_data_0001.csv.1

```diff
@@ -0,0 +1 @@
+morris_0,morris_1,morris_2
```

Comment:
Do we need this guy?

Reply:
Yes, this is to make sure that the sampler values are repeatable in the same simulation.

Reply:
Got it, thanks!